pyaedt/modules/report_templates.py

```diff
@@ -462,6 +462,488 @@ def _context(self):
             ctxt = ["Domain:=", self.domain]
         return ctxt
 
+    @pyaedt_function_handler()
+    def _change_property(self, tabname, property_name, property_val):
+        if not self._is_created:
+            self._post._app.logger.error("Plot has not been created. Create it and then change the properties.")
+            return False
+        arg = [
+            "NAME:AllTabs",
+            ["NAME:" + tabname, ["NAME:PropServers", "{}:{}".format(self.plot_name, property_name)], property_val],
+        ]
+        self._post.oreportsetup.ChangeProperty(arg)
+        return True
+
+    @pyaedt_function_handler()
+    def edit_grid(
+        self,
+        minor_x=True,
+        minor_y=True,
+        major_x=True,
+        major_y=True,
+        style_minor="Solid",
+        style_major="Solid",
+        minor_color=(0, 0, 0),
+        major_color=(0, 0, 0),
+    ):
+        """Edit Plot Grid Settings.
+
+        Parameters
+        ----------
+        minor_x : bool, optional
```

Comment:
I would edit this parameter descriptions but seem to no longer have the button to do so. I guess I'll edit them later. Descriptions of Boolean parameters should be written like this: "Whether to enable the minor X grid. The default is ``True``. Descriptions should also use only an initial capital letter. For example, Minor grid style.